XamlControlsGallery/ControlPages/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,75 @@
+<Page
+    x:Class="AppUIBasics.ControlPages.CornerRadiusPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:Windows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:AppUIBasics"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+    <StackPanel>
+        <TextBlock IsTextSelectionEnabled="True">
+            <LineBreak />
+            <Run Text="For basic UI elements like buttons that users see on screen at all times," />
+            <Run Text="the default radius value is" />
+            <Run FontWeight="bold" Text="2px" />
+            <Run Text="." />
+            <LineBreak />
+            <LineBreak />
+            <Run Text="For transient UI elements that appear on screen temporarily, like MenuFlyout, or elements that overlay other UI, like TabView tabs," />
+            <LineBreak />
+            <Run Text="the default radius value we use for is" />
+            <Run FontWeight="Bold" Text="4px" />
+            <Run Text="." />
+        </TextBlock>
+        <TextBlock>
+            <Run Text="These demos require Windows 10 Version 1809 or later to work properly. To set a conditional namespace, check out our" />
+            <Hyperlink
+                NavigateUri="https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/conditional-xaml#:~:text=%20Conditional%20XAML%20%201%20Conditional%20namespaces.%20To,conditional%20XAML%2C%20but%20you%20can%20also...%20More%20"
+                TextDecorations="None"
+                ToolTipService.ToolTip="https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/conditional-xaml#:~:text=%20Conditional%20XAML%20%201%20Conditional%20namespaces.%20To,conditional%20XAML%2C%20but%20you%20can%20also...%20More%20">
+                <Run Text="docs" />
+            </Hyperlink>
+        </TextBlock>
+        <local:ControlExample
+            x:Name="Example1"
+            HeaderText="A button with CornerRadius"
+            MinimumUniversalAPIContract="1809">
+            <Button Windows10version1809:CornerRadius="5" Content="MyButton" />
+            <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;Button CornerRadius=&quot;5&quot; Content=&quot;MyButton&quot;/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+        <local:ControlExample
+            x:Name="Example2"
+            HeaderText="A button without CornerRadius"
+            MinimumUniversalAPIContract="1809">
+            <Button Windows10version1809:CornerRadius="0" Content="MyButton" />
```

Comment:
Please update the conditional name to be consistent with the other control pages; for example, ComboBox page: https://github.com/microsoft/Xaml-Controls-Gallery/blob/34aa38e4bd11eb71af47f2ab95bb6a538e9f50d3/XamlControlsGallery/ControlPages/ComboBoxPage.xaml#L64